docs/sources/development/contributing.md

```diff
@@ -72,11 +72,11 @@ If your pull request is not accepted on the first try, don't be discouraged! If
 
 We're trying very hard to keep Eclipse SmartHome lean and focused. We don't want it to do everything for everybody. This means that we might decide against incorporating a new feature. However, there might be a way to implement that feature on top of Eclipse SmartHome.
 
-## Discuss your Design on the mailing list
+## Discuss your design in the discussion forum
```

Comment:
Is it not a common practice to write nouns with a first upper case letters in headers? Just a few examples:

https://developer.android.com/guide/topics/resources/runtime-changes.html
https://docs.angularjs.org/guide/controller
http://www.oracle.com/technetwork/java/javase/documentation/index.html